library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1437,6 +1461,33 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       }
     }
 
+    public static final int WAVEFORMAT_SIZE = 18;
+    public static final int WAVE_FORMAT_EXTENSIBLE = 0xFFFE;
+    public static final int WAVEFORMATEX_SIZE = 22 + WAVEFORMAT_SIZE;
+    public static final int GUID_SIZE = 16;
+
+    private static final byte ACM_FORMATTAG_A_PCM_INT_LIT = 0x0001;
+    private static final byte[] A_MS_ACM_GUID_PCM = {   (byte) 0x01, (byte) 0x00, (byte) 0x00, (byte) 0x00,
+            (byte) 0x00, (byte) 0x00, (byte) 0x10, (byte) 0x00,
+            (byte) 0x80, (byte) 0x00, (byte) 0x00, (byte) 0xAA,
+            (byte) 0x00, (byte) 0x38, (byte) 0x9B, (byte) 0x71 };
+
+    public static int parseMsAcmCodecPrivate(ParsableByteArray buffer)
+            throws ParserException {
+
+        int bytesLeft = buffer.bytesLeft();
+        int wFormatTag = buffer.readLittleEndianUnsignedShort();
+        if (wFormatTag == WAVE_FORMAT_EXTENSIBLE && bytesLeft == WAVEFORMATEX_SIZE) {
+
+          buffer.setPosition(WAVEFORMAT_SIZE + 6); // skip waveformat(18), unionsamples(2), channelmask(4)
+          byte[] SubFormat = new byte[GUID_SIZE];
```

Comment:
This would then be:

```
UUID subFormatUuid = new UUID(buffer.readLong(), buffer.readLong());
if (subFormatUuid.equals(A_MS_ACM_GUID_PCM)) {
  ....
}
```